docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -2118,6 +2118,11 @@ tenant_federation:
 # group runtime duration must exceed 50.0% of the evaluation interval.
 # CLI flag: -ruler.independent-rule-evaluation-concurrency-min-duration-percentage
 [independent_rule_evaluation_concurrency_min_duration_percentage: <float> | default = 50]
+
+# (experimental) False to disable writing the result of rule evaluation to
+# ingesters. This option can be used for testing purposes.
```

Comment:
```suggestion
# (experimental) Writes the results of rule evaluation to
# ingesters. Use this option for testing purposes. To disable, set to `false`.
```

Reply:
Done in https://github.com/grafana/mimir/pull/9060/commits/407ec7da36aa06b3cd9ed0627bb40daa9cb82e1a